### PR TITLE
julia 1.x compatibility changes

### PR DESCRIPTION
**Pull request by PinkMushroom**
I fixed the majority of the files, but there are still a couple of broken ones.  However those broken files do NOT appear to be Julia 1.x problems, but rather argument problems possibly with GLFW ???

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,32 @@
...
+pkg>dev https://github.com/Gnimuc/STBImage.jl.git
+
+The following programs still do not work due to unresolved bugs
+            
+    16_frag_reject
+    
+    22_geom_shaders
+    
+    23_tessellation_shaders
+
```
Comment (Gnimuc) [inline README.md]: I'm going to fix these examples in other PRs.

Comment (PinkMushroom) [reply README.md]: great. i'll be interested to see what was going on.

**Comment by Gnimuc** [comment]
hi @PinkMushroom, I just rebased this PR on #6, it's good to merge now.

**Comment by Gnimuc** [comment]
I just found the error you mentioned in the discourse has already been fixed by [this commit]( https://github.com/Gnimuc/Videre/commit/3f32371ac77ef8d1405bc8709efc3aca63d4a911), the problem is I didn't patch all of the examples. 😂

**Comment by PinkMushroom** [comment]
BTW, this is only the 2nd PR I've ever done. lol. Is there anything else I need to do ?
can i just  do a pull on my tree to pick up the merge ?

**Comment by Gnimuc** [comment]
I just merged this master. ;) Thanks for the contribution!

**Pull request merged**